docs/rules/closure-actions.md

```diff
@@ -13,12 +13,14 @@ export default Controller.extend({
 });
 ```
 
+## GOOD
```

Comment:
Can you switch to `###` for the subheaders? Only the main header at the top is supposed to use `##`.